src/index.spec.js

```diff
@@ -273,6 +273,29 @@ const tests = [
     expected:
       "For I killed a man for my wound,\neven a young man for my bruise",
   },
+  {
+    params: {
+      name: "",
+      bookId: "1PE",
+      ref: "1:24",
+      quote:
+        "πᾶσα σὰρξ ὡς χόρτος, καὶ πᾶσα δόξα αὐτῆς ὡς ἄνθος χόρτου. ἐξηράνθη ὁ χόρτος, καὶ τὸ ἄνθος ἐξέπεσεν,",
+      occurrence: 1,
+    },
+    expected: `All flesh {is} like grass,
+and all its glory {is} like the flower of the grass.
+The grass was dried up, and the flower fell off`,
+  },
+  {
+    params: {
+      name: "",
+      bookId: "1CO",
+      ref: "1:2",
+      quote: "τῇ ἐκκλησίᾳ τοῦ Θεοῦ & τῇ οὔσῃ ἐν Κορίνθῳ",
+      occurrence: 1,
+    },
+    expected: "to the church of God that is in Corinth",
```

Comment:
Is returning `to the church & that is in Corinth`, leaving out `of God` and putting the ` & `.

Reply:
If occurrence is -1, which is the default for many test cases, it will return the expected string for some reason.